packages/engine/src/avatar/functions/moveAvatar.ts

```diff
@@ -231,11 +231,14 @@ export const getAvatarCameraPosition = (entity: Entity, offset: Vector3, positio
  * @param camera
  */
 export const alignXRCameraPositionWithAvatar = (entity: Entity, camera: PerspectiveCamera | OrthographicCamera) => {
-  const cameraContainerPos = camera.parent!.position
+  const parent = Engine.camera.parent as any
```

Comment:
No need to use the Engine.camera here, the camera parameter refers to the same object. Better to have less dependency on external variables.